docs/reference/issuers/acme/dns01/azuredns.rst

```diff
@@ -0,0 +1,70 @@
+=========================
+AzureDNS
+=========================
+
+Configuring the AzureDNS DNS-01 Challenge for a Kubernetes cluster requires creating a service principal in Azure.
+
+For security purposes, it is appropriate to utilize RBAC to ensure that you properly maintain access control to your resources in Azure. The service principal that is generated by this tutorial has fine grained access to ONLY the DNS Zone in the specific resource group specified. It requires this permission so that it can read/write the _acme_challenge TXT records to the zone.
+
+To create the service principal:
+
+.. code-block:: bash
+  :linenos:
+
+  AZURE_CERT_MANAGER_SP_NAME=SOME_SERVICE_PRINCIPAL_NAME
+  AZURE_CERT_MANAGER_SP_PASSWORD=SOME_PASSWORD
+  AZURE_CERT_MANAGER_DNS_RESOURCE_GROUP=SOME_RESOURCE_GROUP
+  AZURE_CERT_MANAGER_DNS_NAME=SOME_DNS_ZONE
```

Comment:
Could we add a comment line above each of these to advise users what these should be set to?

I think a number of people may approach this with a very basic understanding (i.e. "I just deployed AKS and want some TLS certs").

Can we add comments e.g.

"Pick a service principal name. This can be anything"
"The DNS zone you want to obtain certificates for"

etc :smile:

Reply:
Also: does this guide configure cert-manager to only be able to obtain certs for a single zone?

Could we add an addendum "allowing cert-manager to issue certificates for MORE than one zone"

Reply:
@munnerz Indeed, it does only configure for one zone. I'm not certain how we could get it to issue for more than one zone.